hbw/app/javascript/packs/hbw/components/form/select.js.jsx

```diff
@@ -138,11 +132,23 @@ modulejs.define('HBWFormSelect',
           return 'white';
         };
 
+        const borderColor = (isFocused) => {
+          if (this.state.error) {
+            return 'red';
+          }
+
+          if (isFocused) {
+            return 'black';
+          }
+
+          return '#dce4ec';
```

Comment:
don't we have name for that? like ```mystic``` http://chir.ag/projects/name-that-color/#DCE4EC?

Reply:
It does not work :(

Also, we will replace all this colours with brand styles soon.